README.md

```diff
@@ -34,9 +34,9 @@ Windows is not tested and not supported for now.
   Python.org downloads since Python 2.7.9)
 
 - Install PostgreSQL 9.5.x of later. (9.6 preferred) 
-  On Debian distros use: `sudo apt-get install postgresql-9.5-dev`
+  On Debian distros use: `sudo apt-get install postgresql-9.5`
```

Comment:
I think we need postgres dev or else compiling psycopg will fail.

Reply:
I searched for postgresql, but did not find any package with the name `postgresql-9.5-dev`. Also for installing postgresql we need to add the repository url, so should we add this to the docs too?